packages/web-extension/test/walletManager/WalletRepository.test.ts

```diff
@@ -206,6 +209,55 @@ describe('WalletRepository', () => {
       ]);
     });
 
+    it('allows creating 1852 and 1854 purpose account with the same index', async () => {
+      const storeSubject = new BehaviorSubject([storedLedgerWallet]);
+      store.observeAll.mockReturnValue(storeSubject.asObservable());
+
+      const standardAccountProps = createAccount(
+        0,
+        storedLedgerWallet.accounts[storedLedgerWallet.accounts.length - 1].accountIndex + 1,
+        KeyPurpose.STANDARD
+      );
+      const standardProps = {
+        ...standardAccountProps,
+        walletId: storedLedgerWallet.walletId
+      };
+
+      const multiSigAccountProps = createAccount(
+        0,
+        storedLedgerWallet.accounts[storedLedgerWallet.accounts.length - 1].accountIndex + 1,
```

Comment:
Please clean this up a little (mostly deduplicate stuff)
```suggestion
      accountIndex
```

Reply:
Fixed on 19c238b